### ISSUE TITLE
Left half loses connectivity when switching devices with KVM on dongle

### ISSUE THREAD

**Issue by Zetaphor** [UltimateHackingKeyboard/firmware#1008]
I'm using the USB dongle with no bridge cable.
I have the dongle connected to a KVM switch.
Sometimes (not every time, but most of the time), when I switch from my Linux device to my Windows device the left half will fail to make the transfer and becomes completely unresponsive and stops responding to keypresses.
It works fine again after hitting the reset button

**Comment by mondalaci** [comment]
The reset button of the left half?

**Comment by Zetaphor** [comment]
Correct, I have to reset the left half and everything starts working again
